<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_select_lay">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>740</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择此层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_display_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_move_up">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>800</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>向上一层</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_move_down">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>860</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>向下一层</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_file">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pointer_x">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>660</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pointer_y">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>700</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>660</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>700</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>740</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pointer_current_value">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>740</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_display_freq" native="true">
    <property name="geometry">
     <rect>
      <x>849</x>
      <y>9</y>
      <width>1061</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_select_distance">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>740</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_file_2">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_zoom_start_distance">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>680</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_zoom_stop_distance">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>680</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_freq_zoom">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
